src/twisted/words/test/test_tap.py

```diff
@@ -12,17 +12,17 @@ class WordsTapTests(unittest.TestCase):
     Ensures that the twisted.words.tap API works.
     """
 
-    PASSWD_TEXT = "admin:admin\njoe:foo\n"
-    admin = credentials.UsernamePassword('admin', 'admin')
-    joeWrong = credentials.UsernamePassword('joe', 'bar')
+    PASSWD_TEXT = b"admin:admin\njoe:foo\n"
+    admin = credentials.UsernamePassword(b'admin', b'admin')
+    joeWrong = credentials.UsernamePassword(b'joe', b'bar')
 
 
     def setUp(self):
         """
         Create a file with two users.
         """
         self.filename = self.mktemp()
-        self.file = open(self.filename, 'w')
+        self.file = open(self.filename, 'wb')
```

Comment:
This should be bytes because [`twisted.cred.checkers.FilePasswordDB` reads its data as bytes](https://github.com/twisted/twisted/blob/trunk/src/twisted/cred/checkers.py#L211), and opening this file as text would mean the contents would be whatever the [system's preferred locale is](https://docs.python.org/3/library/functions.html#open), which might not be ASCII congruent.